app/assets/javascripts/modules/track_search.js

```diff
@@ -0,0 +1,24 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackSearch = function() {
+    var that = this;
+
+    that.start = function(container) {
+
+      var filterSearchInput = container.find('table'),
+          tableInput = container.find('.js-filter-table-input');
+      filterSearchInput.on('keyup', '.js-filter-table-input', trackSearch);
```

Comment:
This will fire an analytics event after every keypress. What's the intention of this tracking? What are you trying to  find out?